simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,24 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    lenLst = len(lst)
+    i = 0
+    listBinary = []
+    carry = 1
+
+    while i < len(lst):
+        if lst[i] + carry == 2:
+            listBinary.append(0)
+            carry = 1
+        else:
+            listBinary.append(lst[i] + carry)
+            carry = 0
+        i += 1
```

Comment:
Is the "i += 1" part of the conditional structure?

Reply:
yeah.

Reply:
Nope, it does not … 

It is part of the loop structure and it do nothing with the conditional structure.